packages/cli/generators/model/index.js

```diff
@@ -154,8 +164,30 @@ module.exports = class ModelGenerator extends ArtifactGenerator {
       },
     ])
       .then(props => {
-        if (typeof props.modelBaseClass === 'object')
-          props.modelBaseClass = props.modelBaseClass.value;
+        if (!this.isBaseClassChecked) {
+          if (typeof props.modelBaseClass === 'object')
+            props.modelBaseClass = props.modelBaseClass.value;
+          // Find whether the specified base class is one of the available base
+          // class list
+          if (
+            !props.modelBaseClass &&
+            !this.isValidBaseClass(
```

Comment:
Please extract this long multi-line condition into a helper variable.

```ts
const isValidBase = this.isValidBaseClass(
  // ...
);

if (!props.modelBaseClass && !isValidBase) {
  // etc.
}
```

See also https://loopback.io/doc/en/contrib/style-guide.html#indentation-of-multi-line-expressions-in-if